3542/1/lib/repo.rb

```diff
@@ -0,0 +1,58 @@
+require 'httparty'
+require 'nokogiri'
+require 'open-uri'
+require_relative 'util'
+
+class Repo
+  URI = 'https://api.github.com/search/repositories'.freeze
+
+  attr_reader :used_by
+
+  def initialize(gem_name)
+    @repo = collect_repository_info_for gem_name
+    @html = Util::Parse::HTML.parse @repo['html_url']
```

Comment:
this line spoils all method style :(
I would move `Util::Parse::HTML.parse @repo['html_url']` to method called something like `parse_html`